docs/_posts/DevinTDHa/2025-08-04-Phi_4_mini_instruct_Q4_K_M_gguf_en.md

```
@@ -0,0 +1,109 @@
+    .setNPredict(20) \
+    .setNGpuLayers(99) \
+    .setTemperature(0.4) \
+    .setTopK(40) \
+    .setTopP(0.9) \
+    .setPenalizeNl(True)
+pipeline = Pipeline().setStages([document, autoGGUFModel])
+data = spark.createDataFrame([["Hello, I am a"]]).toDF("text")
+
+result = pipeline.fit(data).transform(data)
+result.select("completions").show(truncate = False)
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)